int8 result
---
int64 left_damp
int64 right_damp
